Changelog.md

```diff
@@ -1,7 +1,7 @@
 ### 0.7.2 (unreleased)
 
 Language Features:
-
+ * Code generator: Implementing copying arrays of structs to storage.
```

Comment:
Missing changelog entry for copying nested dynamic arrays from calldata to storage